packages/rn-tester/IntegrationTests/LayoutEventsTest.js

```diff
@@ -21,7 +20,7 @@ const React = require('react');
 const ReactNative = require('react-native');
 const deepDiffer = require('react-native/Libraries/Utilities/differ/deepDiffer');
 
-const {Image, LayoutAnimation, StyleSheet, Text, View} = ReactNative;
+const {Image, LayoutAnimation, Platform, StyleSheet, Text, View} = ReactNative; // [macOS Platform]
```

Comment:
nit to follow convention
```suggestion
const {Image, LayoutAnimation, Platform, StyleSheet, Text, View} = ReactNative; // [macOS] 
```
Or if we want to be explicit we added platform
```suggestion
const {Image, LayoutAnimation, Platform, StyleSheet, Text, View} = ReactNative; // [macOS] Platform added
```